extensions/amp-story-dev-tools/0.1/amp-story-dev-tools-tab-logs.css

```diff
@@ -0,0 +1,86 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+amp-story-dev-tools-tab-logs {
+  overflow-y: scroll !important;
+  margin-top: 30px !important;
+  scrollbar-width: none !important;
+  --i-amphtml-story-dev-tools-logs-error-color: #FF5252 !important;
+  --i-amphtml-story-dev-tools-log-message-background: #ffffff0f;
+  --i-amphtml-story-dev-tools-log-spec-white-accent: #ffffff22;
+}
+
+amp-story-dev-tools-tab-logs::-webkit-scrollbar {
+  display: none !important;
+}
+
+amp-story-dev-tools-tab-logs h1 {
+  font-size: 16px;
+}
+
+amp-story-dev-tools-tab-logs h1 * {
+  vertical-align: middle;
+  margin-inline-end: 10px;
+}
+
+.i-amphtml-story-dev-tools-log-message {
+  margin-bottom: 16px !important;
+  background: var(--i-amphtml-story-dev-tools-log-message-background) !important;
+  color: white !important;
+  position: relative !important;
+  padding: 16px !important;
+  border-radius: 16px !important;
+}
+
+.i-amphtml-story-dev-tools-log-type {
+  margin: 0 !important;
+  font-weight: 700 !important;
```

Comment:
How do you know the font for this weight is loaded?

Reply:
We're loading 400 and 700 in `amp-story-dev-tools`